packages/create-graphback/src/init/starterTemplates.ts

```diff
@@ -1,16 +1,17 @@
 import { createWriteStream, mkdirSync, existsSync } from 'fs';
 import chalk from 'chalk';
 import ora from 'ora'
-import * as github from 'parse-github-url';
-import * as request from 'request';
+import gh from 'parse-github-url';
+import request from 'request';
 import * as tar from 'tar';
 import * as tmp from 'tmp';
+import externalTemplates from '../../community-templates.json';
```

Comment:
Getting `Cannot find module '../../community-templates.json'. Consider using '--resolveJsonModule' to import module with '.json' extensionts(2732)`

Reply:
Actually having it as js file inside repository and contribution guide listing this file is much better than root. I have changed typescript config but it seems to be not applied.